Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,903 @@
+            page = 0
+            while True:
+                findings = dspm_get_risk_findings(client, demisto.args(), page)
+                if not findings:
+                    if page == 0:
+                        demisto.info("No risks were fetched")
+                        readable_output = "### Risk Findings\n **No entries.**\n"
+                        return_results(
+                            CommandResults(
+                                outputs_prefix="DSPM.RiskFindings",
+                                outputs_key_field="id",
+                                readable_output=readable_output,
+                            )
+                        )
+                    break  # No more findings to fetch
+
+                return_results(
+                    CommandResults(
+                        outputs_prefix="DSPM.RiskFindings",
+                        outputs_key_field="id",
+                        outputs=findings,
+                    )
+                )
+                page += 1
+        elif demisto.command() == "dspm-get-risk-finding-by-id":
+            return_results(get_risk_finding_by_id(client, demisto.args()))
+        elif demisto.command() == "dspm-get-list-of-assets":
+            page = 0
+            while True:
+                assets = get_list_of_assets(client, demisto.args(), page)
+                if not assets:
+                    if page == 0:
+                        readable_output = "### List of assets\n **No entries.**\n"
+                        return_results(
+                            CommandResults(
+                                outputs_prefix="DSPM.Assets",
+                                outputs_key_field="id",
+                                outputs=[],
+                                readable_output=readable_output,
+                            )
+                        )
+                    break
+
+                return_results(
+                    CommandResults(
+                        outputs_prefix="DSPM.Assets",
+                        outputs_key_field="id",
+                        outputs=assets,
+                    )
+                )
+                page += 1
+        elif demisto.command() == "dspm-get-asset-details":
+            return_results(get_asset_details(client, demisto.args()))
+        elif demisto.command() == "dspm-get-asset-files-by-id":
+            return_results(get_asset_files_by_id(client, demisto.args()))
+        elif demisto.command() == "dspm-get-data-types":
+            return_results(get_data_types(client))
+        elif demisto.command() == "dspm-get-data-types-findings":
+            page = 0
+            all_data_type_findings = []
+
+            while True:
+                data_type_findings = get_data_type_findings(
+                    client, demisto.args(), page
+                )
+                if not data_type_findings:
+                    if page == 0:
+                        readable_output = (
```

Comment:
please use tableToMarkdownm see this article:
https://xsoar.pan.dev/docs/integrations/code-conventions#tabletomarkdown

Reply:
Sure